tests/stress/main_test.go

```diff
@@ -3,22 +3,23 @@ package system
 import (
 	"flag"
 	"fmt"
+	"log"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/quintilesims/layer0/common/logutils"
+	"github.com/quintilesims/layer0/common/logging"
 )
 
 var (
-	dry   = flag.Bool("dry", false, "Perform a dry run - don't execute terraform 'apply' commands")
-	debug = flag.Bool("debug", false, "Print debug statements")
-	log   = logutils.NewStandardLogger("Test")
+	dry    = flag.Bool("dry", false, "Perform a dry run - don't execute terraform 'apply' commands")
+	debug  = flag.Bool("debug", false, "Print debug statements")
+	logger = logging.NewLogWriter(*debug)
```

Comment:
You don't need this variable (also, fwiw, if we did, you would need to call `flag.Parse()` first).

After `flag.Parse()`, you need to do the following:
```
logger := logging.NewLogWriter(debug)
log.SetOutput(logger)
```

and just logging during the tests should be done with `log.Printf("[DEBUG] ...`